src/Mesh/MeshInline.hh

```diff
@@ -369,7 +371,11 @@ minimumScale() const {
       result = std::min(result, (face.position() - zonePosition).magnitude2());
     }
   }
+#ifdef USE_MPI
```

Comment:
I wonder if there is a better pattern that hides all these ifdefs. Perhaps it would be simple enough to write functions like allReduceMax and allReduceMin that do this ifdef pattern behind the scenes.

```
double allReduceMin(double val) {
#ifdef USE_MPI
    return allReduce(val, MPI_MIN, Communicator::communicator());
#else
    return val;
#endif
}
```

Reply:
I think this kind of big change could be left for a future merge request.

Reply:
The `allReduce` interface is already guarded by `USE_MPI`, the changes guarding all of these calls shouldn't be necessary. I think the issue here is that our usual thin interface for `communicator` when building without MPI support is also guarded out. Communicator headers should be included with or without MPI support in order to support the current `allReduce` interface.

Reply:
I've been merging Landon's branch (bugfix/central_mpi_calls) into mine to keep up-to-date.